### DISCUSSION TITLE
모바일 테스트시 발생하는 오류

### DISCUSSION THREAD

**Discussion by jehkael** [naverz/zepeto-studio-kor#1057] [5. Q&A]
1. 멀티플레이 On/Off 확인했습니다.
2. PC 테스트 진행할때는 제대로 작동하고있습니다
![KakaoTalk_20220724_120306964](https://user-images.githubusercontent.com/109885218/180630429-dff3e439-d62d-4c49-80ef-7a1b85e0fd5a.jpg)
.

**Comment by jehkael** [comment]
1. 월드패키지 업데이트로 해당 문제는 해결이 되었지만
2. #945 의 두 번째 오류와 같은 현상이 발생합니다. 방법이 있을까요?
유니티 버전 2020.3.9f1

**Comment by jehkael** [comment]
모바일테스트 구동시 첫 scene이 SampleSceneV2로 설정되는 이유였습니다.
첫 scene을 설정하는 방법이 있을까요?

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, Build Settings에서 설정할 수 있습니다.
아래 공식 가이드 영상에서 참고하여 주세요~
https://youtu.be/trGRK3geZWo

감사합니다.
